pkg/credentialprovider/config.go

```diff
@@ -122,14 +123,27 @@ func ReadDockerConfigJSONFile(searchPaths []string) (cfg DockerConfig, err error
 	if len(searchPaths) == 0 {
 		searchPaths = DefaultDockerConfigJSONPaths()
 	}
+	return ReadSpecificDockerConfigJsonFile(searchPaths, configJsonFileName)
+}
 
+// ReadDockerConfigJsonTransformationFile attempts to read a legacy .dockerconfigjson file from the given paths.
+// if searchPaths is empty, the default paths are used.
+func ReadDockerConfigJsonFileUsedInSecret(searchPaths []string) (cfg DockerConfig, err error) {
+	if len(searchPaths) == 0 {
+		searchPaths = DefaultDockercfgPaths()
+	}
+	return ReadSpecificDockerConfigJsonFile(searchPaths, configJsonFileNameInSecret)
+}
```

Comment:
this wrapper function seems unnecessary, does it add that much benefit over callers just call ReadSpecificDockerConfigJsonFile directly w/ the correct args (perhaps keeping a public configJsonFileNameInSecret constant for them to use)?